priv/repo/migrations/20240509181234_change_days_of_week_column_type.exs

```diff
@@ -0,0 +1,8 @@
+defmodule Screenplay.Repo.Migrations.ChangeDaysOfWeekColumnType do
+  use Ecto.Migration
+
+  def change do
+    execute "ALTER TABLE pa_message ALTER COLUMN days_of_week TYPE integer[] USING (days_of_week::integer[])"
```

Comment:
I think that's ok. This keeps us from needing to switch back and forth from string to int in queries.

Reply:
question: what makes this migration irreversible? Is it just because once there are values in this column using whatever type that it's defined to use, we can't change it anymore?

Reply:
Certain constructs make a migration irreversible, like `execute/1`, because Ecto has no way to know how to undo the described change. You can avoid it with e.g. `execute/2` by supplying the reverse transformation, though in some cases as you mentioned that might require losing some data. In practice this mostly matters in local dev, where you might end up switching branches a lot.